### PR TITLE
subclassing: move parent invocation fnct in Ext trait

### PR DESCRIPTION
**Pull request by fengalin**
This reduces the risk for the user to override them, or even worse, let the user believe that it has an actual effect to override them.

See https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/merge_requests/223#note_112472

### CODE REVIEW

**Comment by sdroege** [comment]
> This reduces the risk for the user to override them.

Or worse, the user believing that it has an actual effect to override them :)

**Comment by fengalin** [comment]
> Or worse, the user believing that it has an actual effect to override them :)

Indeed :) Do you want me to amend the commit message?

File: src/subclass/object.rs [hunk 1]
```diff
@@ -320,7 +306,31 @@ unsafe impl<T: ObjectSubclass> IsSubclassable<T> for ObjectClass {
     }
 }
 
-pub trait ObjectImplExt: ObjectImpl + ObjectSubclass {
+pub trait ObjectImplExt {
+    /// Chain up to the parent class' implementation of `glib::Object::constructed()`.
+    ///
+    /// Do not override this, it has no effect.
```
Comment (sdroege) [inline src/subclass/object.rs]: Well, it can't be overridden anymore so this part of the docs can go away :)

**Comment by sdroege** [comment]
Looks good to me apart from that line in the docs, thanks!

**Comment by sdroege** [comment]
> Indeed :) Do you want me to amend the commit message?

No that's fine

**Comment by fengalin** [comment]
Note that AppVeyor tests are marked as "succeeded" when the 64bits version is still in progress.

File: src/subclass/object.rs (hunk 1 above)
Comment (fengalin) [reply src/subclass/object.rs]: Done! :)

**Comment by sdroege** [comment]
Thanks, will merge once CI is happy again

**Pull request merged**